src/components/Modals/Settings/index.tsx

```diff
@@ -75,10 +75,25 @@ const SettingsModal = ({ CONFIG, updateAppConfig } : Props) => {
           <Button onClick={onBackupClick}>{t("settings.backupBtn")}</Button>
         </div>
       </div>
+      <h2>{t("settings.versionHeading")}</h2>
+      <div className="setting-row">
+        <label className="col description">
+          {t("grid.spicetifyMarketplace")} {MARKETPLACE_VERSION}
+        </label>
+        <div className="col action">
+          <Button onClick={copyVersion}>{t("settings.versionBtn")}</Button>
+        </div>
+      </div>
     </div>
   );
 };
 
+const copyVersion = () => {
+  Spicetify.Platform.ClipboardAPI.copy(MARKETPLACE_VERSION);
+  // TODO: It looks like the notification is underneath the modal, so is not visible
+  Spicetify.showNotification(t("settings.versionCopied"));
```

Comment:
We could change the text on the Copy button instead yeah?

Reply:
Good call. Changed to do this. It'll change back after 3s, which I think makes sense?

Reply:
That or no timeout at all, since I think it's pretty clear for the user that they copied the text already.